library/src/interactor_impl.cxx

```diff
@@ -615,4 +718,49 @@ void interactor_impl::UpdateRendererAfterInteraction()
 {
   this->Internals->Style->UpdateRendererAfterInteraction();
 }
+
+//----------------------------------------------------------------------------
+void interactor_impl::setViewOrbit(const std::string& view)
+{
+  const auto toZup = zUpTransform(self->VTKInteractor);
+  const auto fromZup = toZup->GetInverse();
```

Comment:
not sure transfrom to  and from zUp is needed. wdyt @Meakk ?

Reply:
That one was my suggestion, it's a change of basis from the renderer's `right,forward,up` (which is configurable) to `x,y,z` (and then back) so we can just internally work with `+z` as up regardless of the configured up direction.

The current implementation of this PR could do without this transforms approach (using  `renderer->GetEnvironmentUp()` and `renderer->GetEnvironmentRight()` directly for the direction vectors).

But it may be nice to have the more generic change of basis for some more involved math, eg. if we want to manipulate or animate the camera based on azimuth/elevation (we'd want a reliable z up to compute spherical coordinates so we don't have to do the whole `{vector[axes[0]], vector[axes[1]], vector[axes[2]]}` dance to reorder the axes all the time)